var/spack/repos/builtin/packages/intel-daal/package.py

```diff
@@ -48,11 +41,12 @@ class IntelDaal(IntelInstaller):
 
     provides('daal')
 
-    def install(self, spec, prefix):
-
-        self.intel_prefix = os.path.join(prefix, "pkg")
```

Comment:
OK, now I think I remember. The Intel packages have their own complicated installation structures. If you install directly into "prefix", you will see all of those internals. For daal here's what you get:
```

bash-4.1$ ls /nfs/tmp2/lee218/spack/opt/spack/linux-rhel6-x86_64/gcc-4.8.5/intel-daal-2017.2.174-scj4qxzbtwaqk5clatskmw5bl7chbbz7/pkg/
bin                                 daal                           samples_2017
compilers_and_libraries             documentation_2017             tbb
compilers_and_libraries_2017        intel_sdp_products.db
compilers_and_libraries_2017.2.174  parallel_studio_xe_2017.2.050
```

I think I wanted to hide this complexity under the "pkg" sub directory and then expose the more typical sub directories like bin, lib, include, and such.